SteamKit2/SteamKit2/Steam/WebAPI/WebAPI.cs

```diff
@@ -443,6 +449,16 @@ public override bool TryInvokeMember( InvokeMemberBinder binder, object[] args,
         /// <returns>A dynamic <see cref="Interface"/> object to interact with the Web API.</returns>
         public static Interface GetInterface( Uri baseAddress, string iface, string apiKey = "" )
         {
+            if ( baseAddress == null )
```

Comment:
Shouldn't these be done in `AsyncInterface` constructor instead?

Reply:
Could be, but then param names could get out of sync.